SUPPORT_POLICY.md

```diff
@@ -0,0 +1,25 @@
+Distroless is designed to follow debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
+
+The current estimation of EOL of images
+
+| Image       |  Support    | Debian 10 EOL | Debian 11 EOL   |
+| ----------- | ----------- | ------------- | --------------- |
+| static      | standard    | aug 2022      | debian 12 + 1yr |
+| base        | standard    | aug 2022      | debian 12 + 1yr |
+| cc          | standard    | aug 2022      | debian 12 + 1yr |
+| java        | standard    | jan 2022      | debian 12 + 3mo |
+| node        | standard    | jan 2022      | debian 12 + 3mo |
+| python      | standard    | jan 2022      | debian 12 + 3mo |
+
```

Comment:
I think the language runtimes should be deprecated a bit quicker or the build will balloon like crazy.

Reply:
Why's that?  I think it's worth adding rationale in this doc.

Also worth adding that this affects the `:debian*` tags specifically: the `:latest` tags will just continue.

And what does "standard" entail?  Does it need to be mentioned here?

Reply:
Yeah I can see how "standard" is a bit confusing without context. I might just remove it. It was a reference to [debian's](https://wiki.debian.org/DebianReleases#Production_Releases) provided EOL date (not LTS or ELTS)